griptape/artifacts/video_artifact.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from attrs import define, field
+
+from griptape.artifacts import BlobArtifact
+
+
+@define
+class VideoArtifact(BlobArtifact):
+    """Stores video binary data and relevant metadata.
+
+    Attributes:
+        value: The video binary data.
+        mime_type: The video MIME type.
+        resolution: The resolution of the video (e.g., 1920x1080).
+        duration: Duration of the video in seconds.
+    """
+
+    aspect_ratio: tuple[int, int] = field(default=(16, 9), kw_only=True)
```

Comment:
open question: do providers enumerate possible aspect ratios, or let you do weirdo things